src/tests/ed25519_tests.rs

```diff
@@ -549,3 +550,24 @@ fn test_sk_zeroization_on_drop() {
         unsafe { ::std::slice::from_raw_parts(bytes_ptr, ED25519_PRIVATE_KEY_LENGTH) };
     assert_ne!(sk_memory, &sk_bytes[..]);
 }
+
+#[test]
+fn wycheproof_test() {
```

Comment:
Note that we use zip215 (the cofactored version of ed25519 verification). 

I'm happy that the wycheproof tests work independently on the variation, so I guess they were carefully selected to be agnostic to the ed25519 mode. That said approving this PR.

Note: But if in the future (ie they update their json edge cases) any of these tests fail, we should check that it's not because of our zip215 mode.

Reply:
if we want to be on top of the wycheproof, we need to import the json file from google/wycheproof directly, the wycheproof crate may be lagging behind